aim/web/api/__init__.py

```diff
@@ -19,6 +19,15 @@ def create_app():
     from aim.web.api.projects.views import projects_router
     from aim.web.api.tags.views import tags_router
     from aim.web.api.views import general_router
+
+    # v2 API routes
+    from aim.web.api.v2.views.runs import run_router
+    from aim.web.api.v2.views.tags import tag_router
+    from aim.web.api.v2.views.experiments import experiments_router
+
+    app.include_router(run_router, prefix='/api/v2/runs')
```

Comment:
@mihran113 @alberttorosyan runs used to be named as `commits` in previous versions. Therefore, probably, these blueprints should be merged into one.

Reply:
just didn't want to mess with the existing API so the UI team will have time to and keep frontend functional in all common branches. Indeed we should review/drop old APIs once everything is integrated.

Reply:
@alberttorosyan Fair enough, but currently UI team is not using any API endpoint. 
All endpoint are going to be refactored, based on the new storage.